### PR TITLE
Feature/sub menus

### PR DESCRIPTION
**Pull request by girishranganathan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,3 +40,5 @@ log.txt
 **/_site
 **/.sass-cache
 **/.jekyll-metadata
+
+mesheryctl/.meshery/
```
Comment (leecalcote) [inline .gitignore]: @girishranganathan can we move to line 3 or so under #Meshery?

File: Makefile [hunk 2]
```diff
@@ -38,6 +38,7 @@ run-local:
 	SAAS_BASE_URL="https://meshery.layer5.io" \
 	PORT=9081 \
 	DEBUG=true \
+	ADAPTER_URLS="mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001" \
```
Comment (leecalcote) [inline Makefile]: Are we missing 10002? What is ADAPTER_URLS used for?

File: .gitignore (hunk 1 above)
Comment (girishranganathan) [reply .gitignore]: done

File: Makefile (hunk 2 above)
Comment (girishranganathan) [reply Makefile]: Purely for local dev. This is different from the one used in docker-compose.

**Comment by girishranganathan** [comment]
@leecalcote can you please fix the DCO issue in your commit?

**Pull request merged**